Bottom: 906cd6662b83b0691c6489b81cbe0f40311d9e1b
Top:    aac9bb425053f8aecfa56b7e8ba83ebe965c44bf
Author: Allen Short <ashort@mozilla.com>
Date:   2019-01-28 22:21:22 -0600

Show query results for execution after changing data sources (re #29) (fixes #774)


---
diff --git a/client/app/pages/queries/view.js b/client/app/pages/queries/view.js
index 8d6c6474..88870fef 100644
--- a/client/app/pages/queries/view.js
+++ b/client/app/pages/queries/view.js
@@ -128,6 +128,7 @@ function QueryViewCtrl(
       return;
     }
 
+    $scope.dataSourceChanged = false;
     getQueryResult(0, $scope.selectedQueryText);
     $scope.lockButton(true);
     $scope.cancelling = false;
